public/components/freelancer-create.js

```diff
@@ -81,14 +81,16 @@ class CreationForm extends React.Component {
               </label>
             </div>
 
-            <label className="category-selector">
-              Job category:
+            <div className="category-selector">
+              <span className="bar"></span>
+              <label>
+                Job Category
+              </label>
               <select ref="category">
                 <option value="selected" disabled>Please select a job category</option>
```

Comment:
@Spyridox i know, the hint is not supposed to be selectable, i can put it as such but then i have to do a weird check to see if its a valid category